### ISSUE TITLE
Create color class mixin

### ISSUE THREAD

**Issue by gregrickaby** [WebDevStudios/wd_s#270]

**Comment by carrieforde** [comment]
The initial step of this is complete, and work is done on [feature/create-color-classes-mixin](https://github.com/WebDevStudios/wd_s/tree/feature/create-color-classes-mixin).

I have added a mixin that will generate three types of color presentational classes:
1. `.color-`
2. `.background-`
3. `.fill-`

These classes can be easily added to elements during development to keep us from constantly having to write color declarations (and keep our code DRY!), and more importantly, are super easy to pass to functions. In fact, I have updated the svg function to allow the user to pass a color, which will create a fill class. No more styling the colors separately. 😄 

I have also added a `_colors.scss` to our `base` folder for the actual calling of the mixin and color class creation.

This still requires Sassdoc documentation, before being closed completely.
